packages/graphback-codegen-client/src/plugin/templates/gqlTemplates.ts

```diff
@@ -0,0 +1,202 @@
+import { getFieldName, GraphbackOperationType, ModelDefinition } from '@graphback/core'
+import { isWrappingType, getNamedType, GraphQLType, GraphQLObjectType, isOutputType, isLeafType, isCompositeType, isAbstractType } from 'graphql'
+
+function buildReturnFields(t: GraphQLObjectType) {
+  const fieldsMap = t.getFields();
+
+  return Object.keys(fieldsMap).map(key => {
+    const field = fieldsMap[key];
+    if (isWrappingType(field.type)) {
+      const modelType = getNamedType(field.type);
+      if (isCompositeType(modelType)) {
+        return `${field.name}: {
+        // TODO
+        // buildReturnFields(modelType);
+      } `
+      } else {
+        return field.name;
+      }
+    }
+    else {
+      return field.name;
+    }
+  });
+}
+
+export const fragment = (t: ModelDefinition) => {
```

Comment:
could we just pass the graphql type to these functions? I don't see `t` data being used in them.